sunpy/cm/color_tables.py

```diff
@@ -352,13 +353,13 @@ def lasco_color_table(number):
 # Translated from the JP2Gen IDL SXT code lct_yla_gold.pro.  Might be better
 # to explicitly copy the numbers from the IDL calculation.  This is a little
 # more compact.
-sxt_gold_r = np.concatenate((255.0 * np.array(range(0, 185)) / 185.0,
+sxt_gold_r = np.concatenate((255.0 * np.array(list(range(0, 185))) / 185.0,
                             255 * np.ones(71)))
-sxt_gold_g = 255 * (np.array(range(0, 256)) ** 1.25) / (255.0 ** 1.25)
+sxt_gold_g = 255 * (np.array(list(range(0, 256))) ** 1.25) / (255.0 ** 1.25)
 sxt_gold_b = np.concatenate((np.zeros(185),
-                             255.0 * np.array(range(0, 71)) / 71.0))
+                             255.0 * np.array(list(range(0, 71))) / 71.0))
 
-grayscale = np.array(range(0, 256))
+grayscale = np.array(list(range(0, 256)))
```

Comment:
Same question in these lines above... an in this case, would not be better to use:
```python
np.arange(0, 256)
```
It looks cleaner to me..